base/domain-afr.yml

```diff
@@ -0,0 +1,381 @@
+
+          Hierdie diens is nie 'n plaasvervanger vir professionele mediese advies/ diagnose/ behandeling nie. Jy moet 'n gekwalifiseerde mediese persoon se raad kry rakende jou mediese kondisie/ sorg, veral as jy ernstige simptome begin ontwikkel.
+
+          Jy bevestig dat jy nie a.g.v. hierdie diens enige mediese advies oor behandeling of sorg sal uitstel of verontagsaam nie. Maak staat op hierdie inligting op jou eie risiko.
+
+          Antwoord met een van die woorde in *VETDRUK* om voort te gaan:
+          *AANVAAR*  ✅ om die bepalings en voorwaardes te aanvaar (of antwoord 📌*KIESLYS* om na die hoofkieslys terug te keer).
+
+  utter_ask_province:
+      - text: |
+          n watter Provinsie woon jy tans?
+
+          Antwoord:
+          *1.* OOS-KAAP
+          *2.* VRYSTAAT
+          *3.* GAUTENG
+          *4.* KWAZULU-NATAL
+          *5.* LIMPOPO
+          *6.* MPUMALANGA
+          *7.* NOORDWES
+          *8.* NOORD-KAAP
+          *9.* WES-KAAP
+
+  utter_incorrect_selection:
+    - text: |
+        Hierdie diens werk die beste wanneer jy die opsies wat in *VETDRUK* voorsien word, gebruik.
+
+        📌Antwoord met *KIESLYS* om na die hoofkieslys terug te keer.
+
+  utter_incorrect_location:
+    - text: |
+        If you have typed your address incorrectly, please try again. If you are unable to provide your address, please TYPE the name of your Suburb, Township, Town or Village (or nearest)
+
+  utter_ask_age:
+    - text: |
+        Hoe oud is jy?
+
+        *1.* <18
+        *2.* 18-39
+        *3.* 40-65
+        *4.* >65
+
+  utter_ask_gender:
+    - text: |
+        Dui asb die geslag aan waarmee jy jouself identifiseer:
+
+        Antwoord met
+        *1.* MANLIK
+        *2.* VROULIK
+        *3.* ANDER
+        *4.* VERKIES OM NIE TE SÊ NIE
+
+  utter_ask_location:
+    - text: |
+        Ons moet jou ligging vasstel.
+
+        Deel asb jou ligging 📍 op WhatsApp deur die volgende stappe te volg:
+        - Druk op die aanheggins-ikoon (+ of  📎) aan die onderkant van Whatsapp.
+        - Kies ligging.
+        - Dit sal 'n kaart laat verskyn met 'n speld wat jou huidige ligging aandui. Druk *Send Your Current Location* as dit jou adres is.
+        - Jy kan ook van verskeie nabygeleë liggings kies indien jy dink dat die GPS nie heeltemal reg is nie.
+
+        Of TIK die naam van jou Buurt, Township, Dorp of Village (of naaste?).
+
+  utter_ask_location_confirm:
+    - text: |
+        ⬛⬛⬛⬜⬜⬜⬜⬜⬜⬜⬜
+
+        Ons gebruik Google Location Services om liggings te bevestig. Bevestig asb dat die adres hieronder is reg volgens die inligting wat jy gedeel het:
+        {location}
+
+        Antwoord met
+        *1.* JA
+        *2.* NEE
+
+  utter_ask_medical_condition:
+    - text: |
+        Het jy enige ander reeds bestaande mediese kondisies waarvan ons moet weet?
+
+        Antwoord
+        *1.* JA
+        *2.* NEE
+        *3.* NIE SEKER NIE
```

Comment:
These keywords should be added to the NLU examples, so they get picked up as valid options.